english/story-continues

```diff
@@ -0,0 +1 @@
+The story continues.  You wake up and it was all a dream.  Then, in the corner of your eye you see [a] (http://ancientexplorers.com/wp-content/uploads/sites/6/2015/12/sparkling-blue-universe_131922.jpg)  
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.